hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -101,6 +101,7 @@
   private final transient HoodieIndex<T> index;
   private transient Timer.Context writeContext = null;
   private transient Timer.Context compactionTimer;
+  private transient Timer.Context indexLookupTimer = null;
```

Comment:
nit: if its going to be used for both look up and update, may be name it `indexTimer`?